azure-pipelines.tools.yml

```diff
@@ -2,7 +2,7 @@
 steps:
   - task: NodeTool@0
     inputs:
-      versionSpec: '10.x'
+      versionSpec: '12.x'
```

Comment:
Do we have to switch to 12? Switching in CI increases changes that we'll accidentally break 10, which a lot of people are still using. (Unless we want to make everyone switch to 12--I see 10 is now in "maintenance" status rather than side-by-side LTS with 12.)

Reply:
I'm gonna try going back to 10... I think I had tried this for speed's sake, but I'll just stick with 10 for reasons you raised.